<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#dialogBtn QPushButton{
background-color: rgb(255, 0, 0);
}
</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">
background-color:#eaeced;
border-radius: 10px;
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="geometry">
          <rect>
           <x>400</x>
           <y>90</y>
           <width>391</width>
           <height>611</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#003854;
border-radius:20px;
</string>
         </property>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>50</y>
            <width>51</width>
            <height>41</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background:transparent;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/Icons/Icons/log-in.svg</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>40</y>
            <width>391</width>
            <height>61</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>24</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:#E88D25;
color: white;
border-radius:5px;</string>
          </property>
          <property name="text">
           <string>LOG IN</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLineEdit" name="loginUsername">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>150</y>
            <width>301</width>
            <height>71</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-radius:none;
border-bottom:2px solid rgba(105,118,132,255);
color:#fff;
padding-bottom:7px;
</string>
          </property>
          <property name="placeholderText">
           <string>User Name</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="loginPassword">
          <property name="geometry">
           <rect>
            <x>50</x>
            <y>240</y>
            <width>301</width>
            <height>71</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <italic>true</italic>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:rgba(0,0,0,0);
border:none;
border-radius:none;
border-bottom:2px solid rgba(105,118,132,255);
color:#fff;
padding-bottom:7px;
</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Password</string>
          </property>
         </widget>
         <widget class="QPushButton" name="loginBtn">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>380</y>
            <width>261</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Terminal</family>
            <pointsize>12</pointsize>
            <weight>50</weight>
            <italic>false</italic>
            <bold>false</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
            <kerning>true</kerning>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton#loginBtn{
background-color: qlineargradient(spread:pad,  x1:0 y1:0, x2:1 y2:0, stop:0 rgba(184,20,27,1), stop:1 rgba(0,0,0,1));
color:rgba(255,255,255,230);
border-radius:5px;
}

QPushButton#loginBtn:hover{
background-color:qlineargradient(spread:pad, x1:0,y1:0.505682,x2:1,y2:0.477,stop:0 rgba(22,20,20,1),stop:1  rgba(233,7,15,1));
color:rgba(255,255,255,230);
border-radius:5px;
}
QPushButton#loginBtn:pressed{
padding-left:5px;
padding-top:5px;
background-color:rgba(105,118,132,200);
}</string>
          </property>
          <property name="text">
           <string>Log In</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>520</y>
            <width>261</width>
            <height>21</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: #fff;</string>
          </property>
          <property name="text">
           <string>Forgot your User Name | Password ?</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>140</x>
            <y>550</y>
            <width>141</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <italic>true</italic>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color:#ed1c24;

background:transparent;</string>
          </property>
          <property name="text">
           <string>Contact: Me</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>550</y>
            <width>31</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background:transparent;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="resources.qrc">:/Icons/Icons/phone-forwarded.svg</pixmap>
          </property>
         </widget>
         <widget class="QLabel" name="txtLoginError">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>120</y>
            <width>311</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(223, 0, 0);
background-color:transparent;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="txtUserError">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>190</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color:transparent;
color:red;</string>
          </property>
          <property name="text">
           <string>X</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <widget class="QLabel" name="txtPasswordError">
          <property name="geometry">
           <rect>
            <x>350</x>
            <y>280</y>
            <width>21</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">#txtPasswordError{
color:red;
background-color:transparent;
}</string>
          </property>
          <property name="text">
           <string>X</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
         <zorder>label_2</zorder>
         <zorder>loginUsername</zorder>
         <zorder>loginPassword</zorder>
         <zorder>loginBtn</zorder>
         <zorder>label_3</zorder>
         <zorder>label_5</zorder>
         <zorder>label_4</zorder>
         <zorder>label</zorder>
         <zorder>txtLoginError</zorder>
         <zorder>txtUserError</zorder>
         <zorder>txtPasswordError</zorder>
        </widget>
        <widget class="QWidget" name="headerLogin" native="true">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>821</width>
           <height>71</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:#003854;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="frame_2">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="minimumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>35</width>
                    <height>35</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="pixmap">
                   <pixmap>:/Images/Images/Yamaha-Logo.ico</pixmap>
                  </property>
                  <property name="scaledContents">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="font">
                   <font>
                    <pointsize>12</pointsize>
                    <weight>75</weight>
                    <bold>true</bold>
                   </font>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">color:#E88D25;</string>
                  </property>
                  <property name="text">
                   <string>INVENTORY PROJECT</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="frame_3">
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>499</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="dialogBtn">
            <property name="styleSheet">
             <string notr="true">#closeWindowsBtn:hover{
background-color:red;

}
#minimizeWindowsBtn:hover,#restoreWindowsBtn:hover{
background-color: #24262b;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::StyledPanel</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QPushButton" name="minimizeWindowsBtn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="toolTip">
                <string>Minimize Window</string>
               </property>
               <property name="styleSheet">
                <string notr="true">border-radius:none;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/Icons/Icons/minus.svg</normaloff>:/Icons/Icons/minus.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="restoreWindowsBtn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="toolTip">
                <string>Restore Window</string>
               </property>
               <property name="styleSheet">
                <string notr="true">border-radius:none;</string>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/Icons/Icons/square.svg</normaloff>:/Icons/Icons/square.svg</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="closeWindowsBtn">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="cursor">
                <cursorShape>PointingHandCursor</cursorShape>
               </property>
               <property name="toolTip">
                <string>Close Window</string>
               </property>
               <property name="styleSheet">
                <string notr="true">border-radius:none;</string>
               </property>
               <property name="text">
                <string/>
               </property>
               <property name="icon">
                <iconset resource="resources.qrc">
                 <normaloff>:/Icons/Icons/x.svg</normaloff>:/Icons/Icons/x.svg</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QLabel" name="txtDatabaseconnection">
         <property name="geometry">
          <rect>
           <x>20</x>
           <y>680</y>
           <width>251</width>
           <height>16</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color:#2c313c;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="label_6">
         <property name="geometry">
          <rect>
           <x>30</x>
           <y>260</y>
           <width>341</width>
           <height>251</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap resource="resources.qrc">:/Images/Images/App-Logo-Main.png</pixmap>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
